images/base/Dockerfile

```diff
@@ -34,18 +32,18 @@ RUN chmod +x /usr/local/bin/clean-install
 # Get dependencies
 # The base image already has: ssh, apt, snapd
 # This is broken down into (each on a line):
-# - packages necessary for installing docker
 # - packages needed to run services (systemd)
-# - packages needed for docker / hyperkube / kubernetes components
-# - misc packages (utilities we use in our own tooling)
+# - CRI (containerd)
+# - packages needed for kubernetes components
+# - misc packages kind uses itself
 # Then we cleanup (removing unwanted systemd services)
 # Finally we disable kmsg in journald
 # https://developers.redhat.com/blog/2014/05/05/running-systemd-within-docker-container/
 RUN clean-install \
-      apt-transport-https ca-certificates curl software-properties-common gnupg2 lsb-release \
       systemd systemd-sysv libsystemd0 \
-      conntrack iptables iproute2 ethtool socat util-linux mount ebtables udev kmod aufs-tools \
-      bash rsync \
+      containerd \
+      conntrack iptables iproute2 ethtool socat util-linux mount ebtables udev kmod \
```

Comment:
we need gnupg2 in order to build the node images with apt

```
/kind build node-image --base-image kindest/base:containerd --image kindest/node:containerd --type apt
E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation
(23) Failed writing body
ERRO[22:37:41] Adding Kubernetes apt key failed! exit status 255
ERRO[22:37:41] Image build Failed! exit status 255
Error: error building node image: exit status 255
```

Reply:
alternatively type=apt can install gnupg2 first if we decide not to get rid of it.